index.html

```diff
@@ -142,55 +155,84 @@ <h3 class="project-title"> Multi-Post Stories <br> Gain+Glory</h3>
+                    <button class="resume-btn">Get my resume</button>
                 </div>
             </div>
         </section>
+        <div class="qualifications" >
+            <div class="skills">
+              <img src="assets/languages.png" alt="Skills">
+              <div>
+                  <h2 class="skill-title color1">Languages</h2>
+                  <ul>
+                      <li class="skill-tag">JavaScript</li>
+                      <li class="skill-tag">Ruby</li>
+                      <li class="skill-tag">HTML</li>
+                      <li class="skill-tag">CSS</li>
+                  </ul>
+              </div>
+          </div>
+          <div class="skills">
+              <img src="assets/frameworks.png" alt="Frameworks">
+              <div>
+                  <h2 class="skill-title color1">Frameworks</h2>
+                  <ul>
+                      <li class="skill-tag">React.js</li>
+                      <li class="skill-tag">Ruby on Rails</li>
+                      <li class="skill-tag">RSpec</li>
+                      <li class="skill-tag">Capybara</li>
+                      <li class="skill-tag">Selenium</li>
+                  </ul>
+              </div>
+          </div>
+          <div class="skills">
+              <img src="assets/skills.png" alt="Languages">
+              <div class="skill-tag-container">
+                  <h2 class="skill-title color1">Skills</h2>
+                  <ul>
+                      <li class="skill-tag">Database Management</li>
+                      <li class="skill-tag">Version Control</li>
+                      <li class="skill-tag">CLI</li>
+                      <li class="skill-tag">Web Development</li>
+                      <li class="skill-tag">API Design</li>
+                  </ul>
+              </div>
+          </div>
+      </div>
+        <section class="contact">
+            <div class="contact-heading-container">
+                <h2 class="primary color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            </div>
+            <form class="form" action="https://formspree.io/f/mwkjgvpr"
+            method="POST">
+                <input type="text" class="mobile-form" required maxlength="30" placeholder="Full name">
+                <div class="desktop-form">
+                    <input type="text" name="first_name" required maxlength="30" placeholder="First Name">
+                    <input type="text" name="list_name" required maxlength="30" placeholder="Last Name">
+                </div>
+                <input type="email" name="email" required maxlength="30" placeholder="Email Address">
+                <textarea type="text" name="message" required class="textarea" maxlength="500" placeholder="Hey I'd like to connect too"></textarea>
+                <button type="submit" class="get-in-touch-btn" name="get-in-touch" id="">Get in touch</button>
+            </form>
```

Comment:
- [ ] Please adjust the size of the `get in touch ` button to make the text more legible. Right now, the text is too compressed and hard to read

Reply:
@shella12 Thank you for identifying the problem. I am having a bit problem with this one. The thing is, it is working fine on my machine. I don't know why it isn't working on others. Could you give me any suggestion on that?